lib/pbench/server/cache_manager.py

```diff
@@ -610,58 +511,25 @@ class CacheManager:
 
             /srv/pbench/archive/fs-version-001/
 
-        This directory is linked from /var/www/html/pbench-tarballs for Apache
-        access.
-
         A directory is created under this root for each controller name used by
         a dataset. Within each ARCHIVE controller directory you'll find:
 
             A set of Pbench Agent dataset results, each comprising a ".tar.xz"
             tar archive (conventionally referred to as a "tarball") and a
             ".tar.xz.md5" MD5 file with the same base name.
 
-    INCOMING
-
-        The INCOMING tree is rooted under the Pbench "top dir" public_html
-        directory, conventionally something like
-
-            /srv/pbench/public_html/incoming/
-
-        It's also linked from /var/www/html/incoming.
-
-        Like the ARCHIVE tree, it will contain a directory for each controller;
-        but a controller has a subdirectory for each dataset result, with the
-        root name of the tarball (without the trailing `.tar.xz`), that
-        contains the unpacked contents of the tarball.
-
-        This exists only after the pbench-unpack-tarballs script has run (based
-        on the TO-UNPACK state link).
-
-    RESULTS
-
-        The RESULTS tree is rooted under the Pbench "top dir" public_html
-        directory, conventionally something like
-
-            /srv/pbench/public_html/results/
+    CACHE
 
-        It's also linked from /var/www/html/results.
+        The CACHE tree is rooted under the Pbench "top dir" directory,
+        with the default path:
 
-        This is almost a mirror of the INCOMING tree structure, except that,
-        instead of directories containing unpacked tarballs, the RESULTS tree
-        contains only a symlink for each dataset name to the INCOMING tree's
-        unpacked tarball directory.
+            /srv/pbench/.cache/
```

Comment:
According to `pbench-server-default.cfg` file, shouldn't this be `/srv/pbench/cache`?

Reply:
we can ignore it as well.